src/MiniProfiler.AspNetCore/MiniProfilerMiddleware.cs

```diff
@@ -50,7 +51,8 @@ public class MiniProfilerMiddleware
             if (basePath.StartsWith("~/", StringComparison.Ordinal)) basePath = basePath.Substring(1);
             if (basePath.EndsWith("/", StringComparison.Ordinal) && basePath.Length > 2) basePath = basePath.Substring(0, basePath.Length - 1);
 
-            BasePath = new PathString(basePath);
+            MatchPath = new PathString(basePath);
+            BasePath = new PathString(basePath.EnsureTrailingSlash());
```

Comment:
This may break the share page include or cause a double `//`, in it's current form. Will need to check and sync behavior with both libs.